projects/ngx-diff/package.json

```diff
@@ -23,5 +23,8 @@
   "bugs": {
     "url": "https://github.com/rars/ngx-diff/issues"
   },
-  "homepage": "https://github.com/rars/ngx-diff#readme"
+  "homepage": "https://github.com/rars/ngx-diff#readme",
+  "exports": {
+    "./styles/": "./styles/"
```

Comment:
Exporting directories is deprecated and not supported in node anymore.
Since I am using webpack, it’s not complaining.

So I recommend to export like `./styles/*": "./styles/*.css"` to avoid problems in the future.

Reply:
Thanks for this suggestion, have updated in https://github.com/rars/ngx-diff/pull/81